vcl_templates/_boundary_headers.vcl.erb

```diff
@@ -0,0 +1,16 @@
+# Protect header from modification at the edge of the Fastly network
+# https://developer.fastly.com/reference/http-headers/Fastly-Client-IP
+if (fastly.ff.visits_this_service == 0 && req.restarts == 0) {
+  set req.http.Fastly-Client-IP = client.ip;
+}
+
+# Original client address (e.g. for rate limiting).
+set req.http.True-Client-IP = req.http.Fastly-Client-IP;
```

Comment:
I'm considering tweaking this on staging in future so that we (if we [include original IPs in goreplay requests](https://github.com/buger/goreplay/wiki/Capturing-and-replaying-traffic#tracking-original-ip-addresses)) we can more effectively test changes to our rate limiting policies

(But this looks fine)